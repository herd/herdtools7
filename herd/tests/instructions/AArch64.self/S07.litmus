AArch64 S07
{
0:X0=0; 0:X1=0; 0:X30=P0:L0;
}
    P0           ;
  B L1           ;
L0:              ;
  MOV W0, #1     ;
  RET            ;
L1:              ;
  BL L0          ;
  MOV W1, #1     ;
forall (0:X0=1/\0:X1=1)
