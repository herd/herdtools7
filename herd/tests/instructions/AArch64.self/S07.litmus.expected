Test S07 Required
States 1
0:X0=1; 0:X1=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=1 /\ 0:X1=1)
Observation S07 Always 1 0
Hash=a4121ff3b0515da6b9b8e9aae2074400

